docs/supported_ops.md

```diff
@@ -17305,9 +17305,15 @@ dates or timestamps, or for a lack of type coercion support.
 <td> </td>
 <td><b>NS</b></td>
 <td> </td>
+<<<<<<< HEAD
```

Comment:
Unresolved merge conflict

Reply:
looks like build_all.sh does not update this any longer.